scripts/support/create_ge_images/create_template.sh

```diff
@@ -35,7 +35,7 @@ export TESTSCRIPT=test.sh
 export UPLOAD_FILE=data.tgz
 export IMAGES_DIR=~/create_ge_images/
 export IP=$(nova floating-ip-list | awk '/^\|[ ]+[0-9]+/ { print $2 }')
-
+export FLAVOR=${FLAVOR:-m1.small}
```

Comment:
This is a default value. User can change it if needed, using this change. Or can change the script. This is a shell script and the constants are at the beginning of the file.

Anyway, m1.small is in any default installation. People does not change this. We use this value directly in other scripts even without a constant.